doc/api-extensions.md

```diff
@@ -2322,3 +2322,9 @@ mounted.
 
 The API extension adds indication whether the LXD version is an LTS release.
 This is indicated when command `lxc version` is executed or when `/1.0` endpoint is queried.
+
+## `oidc_groups_claim`
```

Comment:
This should be in the previous commit, and in the format of other API extensions, .e.g. https://github.com/canonical/lxd/commit/841c023dad44872c89cd193e52ae3b98fa8c0334